server/db/controller.js

```diff
@@ -1,10 +1,40 @@
+/* eslint-disable object-shorthand */
+/* eslint-disable func-names */
 
 const connection = require('./index.js');
-const getAll = function(cb){
-    // execute my query to the db
-    connection.query('SELECT * FROM reviews', cb);
-}
+
+const getReviewItems = function (cb) {
```

Comment:
Is this query returning every single review in your database? If so, you may want to return only reviews pertaining to one specific listing. Just something to consider.

Reply:
Well, I have listing and property ids, which I haven't used them for now to keep the current scope limited.But I will include it to all my queries later.